internal/fusefrontend/node_xattr.go

```diff
@@ -50,13 +47,13 @@ func (n *Node) Getxattr(ctx context.Context, attr string, dest []byte) (uint32,
 		var errno syscall.Errno
 		data, errno = n.getXAttr(attr)
 		if errno != 0 {
-			return minus1, errno
```

Comment:
Why not -1 like https://man7.org/linux/man-pages/man2/getxattr.2.html#RETURN_VALUE says?

Reply:
The behavior of the function is described in:
https://github.com/hanwen/go-fuse/blob/756578b6a0b03511fde147b2f23c5b97217ede27/fs/api.go#L312
Returning -1 does not make sense, since it is ignored on errors:
https://github.com/hanwen/go-fuse/blob/756578b6a0b03511fde147b2f23c5b97217ede27/fuse/opcode.go#L294